charts/spv-wallet-stack/charts/spv-wallet/templates/deployment.yml

```diff
@@ -65,11 +65,16 @@ spec:
                   name: {{ .Release.Name }}-postgresql-spv-wallet
                   key: password
           {{- end }}
-            - name: SPVWALLET_PAYMAIL_BEEF_BLOCK_HEADER_SERVICE_AUTH_TOKEN
+          {{- if .Values.config.paymail.beef.block_headers_service_auth_token }}
```

Comment:
I would prefer to: if this value is set then
1. create a secret with this value in `block-headers-service-auth-token` key
2. use that secret instead of the secret below

BTW. doing it like this will allow you to just choose the secret name depending on the existence of this value or not.

Reply:
Setting auth token moved to secrets